techmatters/flex-plugins
# Case List table styling cleanup

Primary reviewer: @GPaoloni 

## Description
- Changed table width to 1280px(https://marvelapp.com/prototype/df7eed1/screen/86806282)
- Changed adding and fonts in filters and content in table headers and rows (https://marvelapp.com/prototype/df7eed1/screen/86806284 and https://marvelapp.com/prototype/df7eed1/screen/86836094(except follow up dialog responsive aligning))
- Modified the structure of the Filters row. 
- Added extra improvements such as wrapping of content within table row cells and Selection content:
<img width="300" alt="Before" src="https://user-images.githubusercontent.com/42842977/170299518-218af106-e91e-4164-8a69-82297dc5c427.png">
<img width="350" alt="After" src="https://user-images.githubusercontent.com/42842977/170299345-318e7e7c-a34e-469b-a205-0b942daa3c97.png">


### Checklist
- [n/a] New tests added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Related Issues
Fixes #[1202](https://bugs.benetech.org/browse/CHI-1202) partially

### Verification steps
- Check the changes made in this branch against https://marvelapp.com/prototype/df7eed1/screen/86806284
https://marvelapp.com/prototype/df7eed1/screen/86806282
https://marvelapp.com/prototype/df7eed1/screen/86836094 (except follow up dialog responsive aligning)

Ultimately, should be close to https://marvelapp.com/prototype/ai502c9/screen/86188497
